pkg/integration/integration.go

```diff
@@ -3,12 +3,23 @@ package integration
 import (
 	"fmt"
 	"os"
+	"strings"
 
 	color "github.com/logrusorgru/aurora"
 )
 
 func Print() {
-	fmt.Println(bashSource)
+	var currentShell = os.Getenv("SHELL")
```

Comment:
Just an idea, we could warn in case the value is empty, since it would be a very special case of "unknown shell". 
That may help someone debug their environment someday.